### PR TITLE
add edit review and community post

### LINKED ISSUE #84: Edit review and community post

### PR DESCRIPTION
**Pull request by DavidFonsek**
GG

### CODE REVIEW

File: app/Http/Controllers/CommunityPostController.php
Comment (mstermigol) [inline app/Http/Controllers/CommunityPostController.php:1]: Remember to use lang

File: app/Http/Controllers/ReviewController.php
Comment (mstermigol) [inline app/Http/Controllers/ReviewController.php:1]: Remember lang and be carefull when copy pasting from another controller

File: resources/views/communitypost/show.blade.php
Comment (mstermigol) [inline resources/views/communitypost/show.blade.php:1]: Remember to use lang on the onClick

File: app/Http/Controllers/CommunityPostController.php
Comment (DavidFonsek) [reply app/Http/Controllers/CommunityPostController.php:1]: mano lang en que?

Comment (mstermigol) [inline app/Http/Controllers/CommunityPostController.php:1]: Use trans instead of __

File: app/Http/Controllers/ReviewController.php
Comment (mstermigol) [inline app/Http/Controllers/ReviewController.php:1]: Use getters in the update method

File: app/Http/Controllers/TravelController.php
Comment (mstermigol) [inline app/Http/Controllers/TravelController.php:1]: Use trans() instead of __

File: resources/views/communitypost/edit.blade.php
Comment (mstermigol) [inline resources/views/communitypost/edit.blade.php:1]: Dont use styles tag

**Pull request merged**